packages/core/src/components/table/table-component-expandable-rows.stories.tsx

```diff
@@ -48,6 +48,16 @@ export default {
         defaultValue: { summary: false },
       },
     },
+    expanded: {
+      name: 'Expanded',
+      description: 'Expands row when set to true. This is set on row level.',
+      control: {
+        type: 'boolean',
+      },
+      table: {
+        defaultValue: { summary: false },
+      },
```

Comment:
I am not sure if we should have defaultValue. Only reason is that expand prop will then appear in all docs when users copy/paste it and I wonder will it confuse them. What do you think?